pages/docs/[[...page]].tsx

```diff
@@ -44,12 +44,14 @@ const { getStaticPaths, getStaticProps } = getStaticGenerationFunctions(
           return `https://github.com/hashicorp/${PRODUCT.slug}/blob/main/website/${filepath}`
         },
         remarkPlugins: (params) => [
+          ...remarkPlugins,
           remarkRewriteAssets({
             product: PRODUCT.slug,
             version: process.env.CURRENT_GIT_BRANCH,
             getAssetPathParts: (nodeUrl) => ['website', nodeUrl],
           }),
         ],
+        rehypePlugins,
```

Comment:
🚧 Potentially blocking: On https://terraform-website-git-kevin-upgrade-docs-page-hashicorp.vercel.app/docs/glossary , jump links don't seem to be working. I think we're missing `remarkPlugins` & `rehypePlugins` in the `strategy: remote` section below?